tests/languages.rs

```diff
@@ -8,7 +8,7 @@ fn test_swift() {
         "swift_project",
         r#"
             $ tree-sitter-grep -q '(value_argument) @c' --language swift
-            example.swift:2:    atPath: "native"
+            example.swift:    atPath: "native"
```

Comment:
Most of the existing test output changed because (a) those tests are running in "non-terminal-output" mode and (b) now in that mode we don't show line numbers by default (like `ripgrep`)